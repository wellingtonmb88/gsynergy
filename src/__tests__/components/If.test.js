import React from "react";
import { shallow } from "enzyme";
import If from "../../components/If";

describe("render the ProductListScreen", () => {
  it("renders without crashing", () => {
    const tree = shallow(<If />);
    expect(tree).toMatchSnapshot();
  });

  it("testing the rest of the conditional", () => {
    const tree = shallow(
      <If test={true}>
        <span>Test</span>
      </If>
    );
    expect(tree).toMatchSnapshot();
  });
});
