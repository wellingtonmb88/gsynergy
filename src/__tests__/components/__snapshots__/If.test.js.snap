// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the ProductListScreen renders without crashing 1`] = `""`;

exports[`render the ProductListScreen testing the rest of the conditional 1`] = `
<span>
  Test
</span>
`;
